### PR TITLE
[it] Localizzazione Serverless

### LINKED ISSUE #1654: [it] Localizzazione Serverless
https://glossary.cncf.io/serverless/

### PR DESCRIPTION
**Pull request by ugho16**
### Describe your changes
Added Serverless Italian localization

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1654 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/serverless.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Status: Completed
+Category: Tecnologia
+tags: ["architettura", "", ""]
+---
+
+## Cos'è
+
+Il serverless è un modello di sviluppo cloud native che permette
+di costruire ed eseguire applicazioni senza la necessità di gestire i server.
+I server sono comunque presenti nel serverless, ma sono [astratti](/it/abstraction/) dallo sviluppo dell'app.
```
Comment (annalisag-spark) [inline content/it/serverless.md:12]:
```suggestion
I server rimangono presenti nel serverless, ma sono [astratti](/it/abstraction/) dallo sviluppo dell'app.
```

File: content/it/serverless.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Un cloud provider gestisce regolarmente il provisioning, la manutenzione e la [scalabilità](/it/scalability/) dell'infrastruttura dei server.
```
Comment (annalisag-spark) [inline content/it/serverless.md:13]:
```suggestion
Un cloud provider gestisce regolarmente il provisioning, la manutenzione e la [scalabilità](/it/scalability/) dell'infrastruttura server.
```

File: content/it/serverless.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+I team di sviluppo possono semplicemente accorpare il loro codice all'interno di [container](/it/container/) per il deploy.
```
Comment (annalisag-spark) [inline content/it/serverless.md:14]:
```suggestion
I team di sviluppo possono semplicemente impacchettare codice in[container](/it/container/) per la distribuzione.
```

Comment (annalisag-spark) [reply content/it/serverless.md:14]: una proposta, non so se effettivamente _impacchettare_ sia corretto :)

File: content/it/serverless.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+Una volta eseguito il deploy, le applicazioni serverless rispondono al traffico e possono essere automaticamente [scalate verticalmente](/it/vertical-scaling/) a seconda della necessità.
```
Comment (annalisag-spark) [inline content/it/serverless.md:15]:
```suggestion
Una volta distribuite, le applicazioni serverless rispondono alla domanda e [scalano automaticamente](/it/vertical-scaling/) in base alle necessità.
```

File: content/it/serverless.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+Le offerte serverless proposte dai fornitori di cloud pubblico sono generalmente misurate on-demand attraverso un modello di esecuzione ad eventi.
```
Comment (annalisag-spark) [inline content/it/serverless.md:16]:
```suggestion
Le offerte serverless proposte dai fornitori di cloud pubblico sono generalmente misurate on-demand attraverso un modello di esecuzione basato sugli eventi.
```

File: content/it/serverless.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+Di conseguenza, quando una funzione serverless non è in esecuzione, non ha costi.
+
+## Quali problematiche affronta
+
+In un modello standard di [cloud computing](/it/cloud-computing/) di tipo [Infrastructure-as-a-Service (IaaS)](/it/infrastructure-as-a-service/),
+gli utenti acquistano in anticipo unità di capacity, nel senso che pagano un fornitore di cloud pubblico per componenti server sempre attivi che mantengano in esecuzione le applicazioni.
```
Comment (annalisag-spark) [inline content/it/serverless.md:22]:
```suggestion
gli utenti acquistano in anticipo unità di capacità, ovvero pagano un fornitore di cloud pubblico per avere componenti server sempre attivi che mantengano in esecuzione le applicazioni.
```

File: content/it/serverless.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+È compito dell'utente scalare verso l'alto la capacity del server nei momenti di traffico elevato
+e di scalare verso il basso quella stessa capacity quando non è più necessaria.
```
Comment (annalisag-spark) [inline content/it/serverless.md:24]:
```suggestion
e di scalarla verso il basso quando non è più necessaria.
```

File: content/it/serverless.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## In che modo aiuta
+
+Con un'architettura serverless, invece, le applicazioni vegnono avviate soltanto quando serve.
+Quando un evento attiva l'esecuzione del codice dell'applicazione, il fornitore di cloud pubblico alloca delle risorse per quel codice in modo dinamico.
+L'utente smette di pagare nel momento in cui il codice finisce di essere eseguito.
+Oltre ai benefici legati al costo e all'efficienza,
+il serverless libera gli sviluppatori dalle attività ripetitive e secondarie legate alla scalabilità dell'applicazione e al provisioning del server.
+Grazie al serverless, attività ripetitive come la gestione del sistema operativo e dei file di sistema, le patch di sicurezza,
+il load balancing, la gestione della capacity, la scalabilità, il logging e il monitoraggio vengono tutte demandate al fornitore di servizi cloud.
```
Comment (annalisag-spark) [inline content/it/serverless.md:35]:
```suggestion
il bilanciamento, la gestione della capacità, la scalabilità, il logging e il monitoraggio vengono tutte demandate al fornitore di servizi cloud.
```

**Review by annalisag-spark — APPROVED** [review]

**Review by fsbaraglia — APPROVED** [review]
good one

**Pull request merged**